frontend/src/components/common/PageLoading.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="loading-overlay">
+    <div class="loading-background"></div>
```

Comment:
Please don't usr this loader. It is not companies style. Let's use loader zone inside page